### PR TITLE
on-premise python docs

### PR DESCRIPTION
**Pull request by jfarrimo**
Documentation for the tweaks necessary to use the python client with an on-premise installation.

### CODE REVIEW

File: on-premise/python.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+you use for the particular python environment you're running in,
+eg. Django, Pyramid, Flask, etc...  Here's an example of the changes
+needed for a couple of common cases.  You can adapt this to your
+particular environment.
+
+## generic python client
+
+The new settings will go into your init call:
+
+```rollbar.init('POST_SERVER_ITEM_ACCESS_TOKEN',
```
Comment (brianr) [inline on-premise/python.md]: `rollbar.init()` takes two positional arguments; the second one (missing here) is environment. Can you add that?

Comment (jfarrimo) [inline on-premise/python.md]: fixed

**Comment by brianr** [comment]
lgtm

**Comment by coryvirok** [comment]
:+1:

**Pull request merged**